src/pl/plpython/expected/plpython_returns_3.out

```diff
@@ -0,0 +1,3064 @@
+--------------------------
+ (value,4)
+(1 row)
+
+SELECT * FROM test_return_table_record('{"first": "value", "second": 4}');
+ first | second 
+-------+--------
+ value |      4
+(1 row)
+
+SELECT test_return_table_record('{"first": "value", "second": 4}')
+FROM gp_single_row;
+ test_return_table_record 
+--------------------------
+ (value,4)
+(1 row)
+
+SELECT (test_return_table_record('{"first": "value", "second": 4}')).*;
+ first | second 
+-------+--------
+ value |      4
+(1 row)
+
+SELECT (test_return_table_record('{"first": "value", "second": 4}')).*
+FROM gp_single_row;
+ first | second 
+-------+--------
+ value |      4
+(1 row)
+
+-- Error conditions
+-- [ERROR] From Python bool
+SELECT test_return_table_record('True');
+ERROR:  attribute "first" does not exist in Python object
+SELECT * FROM test_return_table_record('True');
+ERROR:  attribute "first" does not exist in Python object
+-- [ERROR] From Python empty string
+SELECT test_return_table_record('""');
+ERROR:  malformed record literal: ""
+SELECT * FROM test_return_table_record('""');
+ERROR:  malformed record literal: ""
+-- [ERROR] From Python string
+SELECT test_return_table_record('"value"');
+ERROR:  malformed record literal: "value"
+SELECT * FROM test_return_table_record('"value"');
+ERROR:  malformed record literal: "value"
+-- [ERROR] From Python non-empty list, wrong number of columns
+SELECT test_return_table_record('[False]');
+ERROR:  length of returned sequence did not match number of columns in row
+SELECT * FROM test_return_table_record('[False]');
+ERROR:  length of returned sequence did not match number of columns in row
+-- [ERROR] From Python non-empty list, wrong datatypes
+SELECT test_return_table_record('[4, "value"]');
+ERROR:  invalid input syntax for integer: "value"
+SELECT * FROM test_return_table_record('[4, "value"]');
+ERROR:  invalid input syntax for integer: "value"
+-- [ERROR] From Python empty dict
+SELECT test_return_table_record('{}');
+ERROR:  key "first" not found in mapping
+SELECT * FROM test_return_table_record('{}');
+ERROR:  key "first" not found in mapping
+-- [ERROR] From Python without the needed columns
+SELECT test_return_table_record('{None: None}');
+ERROR:  key "first" not found in mapping
+SELECT * FROM test_return_table_record('{None: None}');
+ERROR:  key "first" not found in mapping
+-- Questionable Success conditions
+--  From Python String (one character per column)
+SELECT test_return_table_record('"a4"');
+ERROR:  malformed record literal: "a4"
+SELECT * FROM test_return_table_record('"a4"');
+ERROR:  malformed record literal: "a4"
+-- From Python Dictionary, with extra fields
+SELECT test_return_table_record('{"first": "value", "second": 4, "third": "ignored"}');
+ test_return_table_record 
+--------------------------
+ (value,4)
+(1 row)
+
+SELECT * FROM test_return_table_record('{"first": "value", "second": 4}, "third": "ignored"}');
+ERROR:  SyntaxError: invalid syntax (<string>, line 1) (plpy_elog.c:106)
```

Comment:
This doesn't work on my computer. It should not be a big deal as long as it passes on our CI.

```
❯ python3 --version
Python 3.10.2
```

```
--- \/home\/cc\/repo\/gpdb6\/src\/pl\/plpython3\/expected\/python3\/plpython_returns\.out       2022-03-25 20:19:38.405854360 +0800
+++ \/home\/cc\/repo\/gpdb6\/src\/pl\/plpython3\/results\/python3\/plpython_returns\.out        2022-03-25 20:19:38.424854218 +0800
@@ -1977,7 +1977,7 @@
 (1 row)

 SELECT * FROM test_return_table_record('{"first": "value", "second": 4}, "third": "ignored"}');
-ERROR:  SyntaxError: invalid syntax (<string>, line 1)
+ERROR:  SyntaxError: unmatched '}' (<string>, line 1)
 -- =====================================================
 -- TEST 4:  RETURN VALUE TESTING - SETOF Compound values
 -- =====================================================
@@ -2093,7 +2093,7 @@
 SELECT test_return_setof_type_record('[{"first": "value", "second": 4, "third": "ignored"}]');

 SELECT * FROM test_return_setof_type_record('[{"first": "value", "second": 4}, "third": "ignored"}]');
-ERROR:  SyntaxError: invalid syntax (<string>, line 1)
+ERROR:  SyntaxError: closing parenthesis '}' does not match opening parenthesis '[' (<string>, line 1)
 --
 -- Case 2: From a setof Table Type
 --
@@ -2200,7 +2200,7 @@
 SELECT test_return_setof_table_record('[{"first": "value", "second": 4, "third": "ignored"}]');

 SELECT * FROM test_return_setof_table_record('[{"first": "value", "second": 4}, "third": "ignored"}]');
-ERROR:  SyntaxError: invalid syntax (<string>, line 1)
+ERROR:  SyntaxError: closing parenthesis '}' does not match opening parenthesis '[' (<string>, line 1)
 -- =====================================================
 -- TEST 5:  RETURN VALUE TESTING - OUT Parameters
 -- =====================================================
@@ -2896,7 +2896,7 @@
 (1 row)

 SELECT * FROM test_return_out_setof_record('[{"first": "value", "second": 4}, "third": "ignored"}]');
-ERROR:  SyntaxError: invalid syntax (<string>, line 1)
+ERROR:  SyntaxError: closing parenthesis '}' does not match opening parenthesis '[' (<string>, line 1)
 -- From Python List of String (one character per column)
 SELECT test_return_out_setof_record('["a4"]');
 ERROR:  malformed record literal: "a4"

======================================================================

```